### PR TITLE
CDPS-44 - Personal page personal details card

### PR DESCRIPTION
**Pull request by danielburnley**

### CODE REVIEW

File: server/services/personalPageService.ts [hunk 1]
```diff
@@ -10,36 +12,64 @@ export default class PersonalPageService {
   }
 
   public async get(prisonerData: Prisoner): Promise<PersonalPage> {
-    // const inmateDetail = await this.prisonApiClient.getInmateDetail(prisonerData.bookingId)
-    const toFullName = (firstName: string, middleNames: string, lastName: string) =>
-      [firstName, middleNames, lastName].filter(s => s !== undefined).join(' ')
+    const inmateDetail = await this.prisonApiClient.getInmateDetail(prisonerData.bookingId)
```
Comment (davidmiddletonmoj) [inline server/services/personalPageService.ts:15]: Could do these in parallel with `Promise.all` to improve performance

File: server/views/partials/personalPage/languages.njk [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% from "../../macros/summaryListRow.njk" import summaryListRow %}
+{% macro otherLanguagesText(language) %}
+  {% if language.canSpeak and language.canWrite %}
```
Comment (davidmiddletonmoj) [inline server/views/partials/personalPage/languages.njk:3]: Do we need to have a `and not <the_other_canSomething>` for the first 3 conditions? i.e. what if all 3 are `true`?

**Review by davidmiddletonmoj — APPROVED** [review]
Some minor comments, but approved to keep things moving

File: server/views/partials/personalPage/languages.njk (hunk 2 above)
Comment (danielburnley) [reply server/views/partials/personalPage/languages.njk:3]: Ah yeah you're right - I'll add that in in the next PR

**Pull request merged**